lib/helper.js

```diff
@@ -9,6 +11,30 @@ function typeBasedParser(type,data) {
 		return null
 }
 
+function readFile(dir, filename, callback) {
+	//console.log('Dir', dir);
+	filename = typeof(filename) === 'string' && filename.length > 0 ? filename : false;
+
+	if(filename) {
+	   fs.readFile(`${dir}/${filename}`, 'utf8', (err, file) => {
+          if(!err && file && file.length > 0 ) {
+			// convert file from buffer to string if the file is an html file
+			if(filename.indexOf('.html') || file.indexOf('.htm')) file = file.toString();
+
+			// console.log('File content', file);
+			callback(false, file);
+		  } else {
+			// console.log('readfile error', err);
+			callback('cannot read file')
```

Comment:
Please consider an error object and pass this as message, so that stack tracks can be generated. 
something like 
callback(new Error(<message goes in here>));